### PR TITLE
Create wulfland.md

### PR DESCRIPTION
**Pull request by wulfland**

### CODE REVIEW

File: docs/_slides/wulfland.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Greetings from wulfland!
```
Comment (wulfland) [inline docs/_slides/wulfland.md:6]: ❓ What do you think, @kaufm?

Comment (kaufm) [reply docs/_slides/wulfland.md:6]:
```suggestion
Greetings from @wulfland and @kaufm!
```

File: docs/_slides/wulfland.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+![dinotocat](https://octodex.github.com/images/dinotocat.png)
+{: .center}
```
Comment (kaufm) [inline docs/_slides/wulfland.md:9]:
```suggestion
![Daftpunktocat-Thomas](https://octodex.github.com/images/daftpunktocat-thomas.gif)
{: .center}
```

File: docs/_slides/wulfland.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+layout: slide
+title: "Welcome to our slide deck!"
+---
+
+Greetings from @wulfland and @kaufm!
```
Comment (kaufm) [inline docs/_slides/wulfland.md:6]: 👍 Looks good

File: docs/_slides/wulfland.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+![Daftpunktocat-Thomas](https://octodex.github.com/images/daftpunktocat-thomas.gif)
+{: .center}
```
Comment (kaufm) [inline docs/_slides/wulfland.md:9]: 🏆

**Review by kaufm — APPROVED** [review]
Looks great 🏅

**Review by kaufm — APPROVED** [review]

**Pull request merged**